src/commands/uninstall.rs

```diff
@@ -32,9 +34,25 @@ pub fn uninstall(options: UninstallOpt) -> anyhow::Result<()> {
     // returned bool indicates if there was any to the lockfile. If this pacakge is uninstalled,
     // there will be a diff created, which causes update to return true. Because no other change
     // is made, we can assume any change resulted in successfully uninstalled package.
-    let result = dataflow::update(vec![], uninstalled_package_names, dir)?;
+    let result = dataflow::update(vec![], uninstalled_package_names, dir.clone())?;
 
-    if !result {
+    // Uninstall the package from /tmp/wax/...
+    let mut wax_uninstalled = false;
+    let mut wax_index = wax_index::WaxIndex::open()?;
+    if wax_index.search_for_entry(options.package.clone()).is_ok() {
+        wax_index.remove_entry(options.package.as_str())?;
+        wax_index.save()?;
+        wax_uninstalled = true;
+    }
```

Comment:
I don't think we should have wax uninstall... We should have simply a way to run wax with no cache.

```
wax --no-cache python
```

Reply:
@syrusakbary I am heavily against that because at the very least I'd need the tool to be able to clean up after itself (to remove all remaining / installed artifacts after installation).

Reply:
also it's just really user-unfriendly not to be able to install packages